discovery-infra/test_infra/logs_utils.py

```diff
@@ -10,7 +10,8 @@
 OC_DOWNLOAD_LOGS_INTERVAL = 10 * 60
 NUM_OF_RETRIES = 6
 
-def verify_logs_uploaded(cluster_tar_path, expected_min_log_num, installation_success):
+
+def verify_logs_uploaded(cluster_tar_path, expected_min_log_num, installation_success, check_oc=False):
```

Comment:
`check_oc` parameter is redundant, you can get the same logic by using `installation_success`

Reply:
these parameters are not identical. installation_success can be false from numerous reasons. only certain failures, or more precisely failure stages indicates that we might have must-gather logs